src/BiometryService.SampleApp.Uno/BiometryService.SampleApp.Uno.Droid/BiometryService.SampleApp.Uno.Droid.csproj

```diff
@@ -17,7 +17,7 @@
     <AndroidUseAapt2>true</AndroidUseAapt2>
```

Comment:
I wonder whether the sample app should be outside the src folder and have its own samples folder isntead?
https://medium.com/code-factory-berlin/github-repository-structure-best-practices-248e6effc405

Reply:
Yes why not.